addons/website/static/src/interactions/cookies/cookies_approval.js

```diff
@@ -38,8 +38,7 @@ export class CookiesApproval extends Interaction {
                 ? ""
                 : "my-3",
         });
-        this.insert(optionalCookiesWarningEl, this.iframeEl, "afterend");
-        this.services["public.interactions"].startInteractions(optionalCookiesWarningEl);
+        this.insert(getCookiesWarningEl(), this.iframeEl, "afterend");
```

Comment:
Something is wrong here... renderAt above should already have inserted in the right location.